watershed_workflow/land_cover_properties_Soumendra.py

```diff
@@ -0,0 +1,328 @@
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import logging
+import math
+import scipy
+
+import rasterio
+import rasterio.transform
+import rasterio.features
+
+import shapely
+
+import watershed_workflow.config
+import watershed_workflow.triangulation
+import watershed_workflow.warp
+import watershed_workflow.plot
+import watershed_workflow.river_tree
+import watershed_workflow.split_hucs
+import watershed_workflow.hydrography
+import watershed_workflow.sources.utils
+import watershed_workflow.sources.manager_shape
+import watershed_workflow.utils
+
+
+lc_type1_colors = {
```

Comment:
I think these are in sources/manager_nlcd.py -- please use those.  If MODIS has its own scheme, we should put that scheme in  manager_modis_appears.py